src/GoodsList.tsx

```diff
@@ -5,10 +5,20 @@ type Props = {
   goods: Good[]
 };
 
+const style = (goodColor: string) => {
+  return ({
+    color: `${goodColor}`,
+  });
+};
+
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
+        style={style(good.color)}
```

Comment:
it can be added in a simpler way like style={{ color: good.color }}

Reply:
^^^ Good suggestion, operation is simple enough to avoid creating extra callbacks.